packages/mobile/src/screens/root-screen/RootScreen.tsx

```diff
@@ -60,6 +63,10 @@ export const RootScreen = ({ isReadyToSetupBackend }: RootScreenProps) => {
     }
   }, [accountStatus])
 
+  useBecomeReachable(() => {
+    if (isReadyToSetupBackend && !isBackendSetup) dispatch(setupBackend())
```

Comment:
I don't think it would matter but we could also add the `!isBackendSetup` condition to the useEffect that does the initial backend setup above. Maybe we could break into a helper function because both places are doing the same thing?